nautobot/extras/templates/extras/objectchange.html

```diff
@@ -95,6 +95,20 @@
                     </tr>
                 </table>
             </div>
+            <div class="panel panel-default">
+                <div class="panel-heading">
+                    <strong>Object Data</strong>
+                </div>
+                <div class="panel-body">
+                    <div class="editor-container" 
+                         data-lang="json"
+                         data-value="{{ object.object_data|render_json:False }}"
+                         style="max-height: 300px; overflow-y: auto;">
```

Comment:
Can we add these styles (at least the `overflow-y: auto;`?) to a proper CSS include file rather than declaring them inline? Just trying to minimize the future work needed if/when we add a more strict CSP to Nautobot.